python/sdk/requirements.txt

```diff
@@ -8,13 +8,15 @@ dataclasses-json>=0.5.2  # allow Flyte version 1.2.0 or above to import Merlin S
 docker<=6.1.3
 GitPython>=3.1.40
 google-cloud-storage>=1.19.0
-protobuf>=3.12.0,<5.0.0  # Determined by the mlflow dependency
+protobuf>=4.21.6,<5
 mlflow==1.26.1
 PyPrind>=2.11.2
 python_dateutil>=2.5.3
 PyYAML>=5.4
 six>=1.10
 urllib3>=1.26
-numpy<=1.23.5  # Temporary pin numpy due to https://numpy.org/doc/stable/release/1.20.0-notes.html#numpy-1-20-0-release-notes
-caraml-auth-google==0.0.0.post7
-pydantic==2.5.3
\ No newline at end of file
+numpy>=1.26.4
```

Comment:
@naufalandika Sorry but it seems like this update to the `numpy` version broke the tests for the other components (`batch-predictor`, `pyfunc-server` and `observation-publisher` ) - see this [run](https://github.com/caraml-dev/merlin/actions/runs/14589346804) for an example. What was the motivation to bump numpy? 😮